src/Controller/Controller.php

```diff
@@ -368,7 +383,6 @@ public function setRequest(Request $request)
     {
         $this->request = $request;
         $this->plugin = isset($request->params['plugin']) ? $request->params['plugin'] : null;
-        $this->view = isset($request->params['action']) ? $request->params['action'] : null;
```

Comment:
Yup it won't, it's now set in render() instead. Using `$this->getView()->view` here would again cause view instance to be created pretty early when rendering might not be even needed.